### PR TITLE
taras quiz....take 2

### PR DESCRIPTION
**Pull request by taramason**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ funtion multiply(x) (
 
 -- YOUR ANSWER HERE --
 
+function multiply(x) {}
+  return x * y;
+}
+
```
Comment (mesprague) [inline README.md:16]: You have an extra curly brace. I'm assuming its a typo :)

File: README.md [hunk 2]
```diff
@@ -37,6 +45,10 @@ var add = function (x, y) {
 
 -- YOUR ANSWER HERE --
 
+- ES5
+- Is the function anonymous? NO
```
Comment (mesprague) [inline README.md:54]: It is anonymous. This is a **function expression**

```
var add = function addition(x,y) {
    return x+y;
}
```

Would be the non-anonymous equivalent

File: README.md [hunk 3]
```diff
@@ -47,6 +59,10 @@ var add = (x, y) => {
 
 -- YOUR ANSWER HERE --
 
+- ES6
+- Is the function anonymous? NO
```
Comment (mesprague) [inline README.md:68]: **function expression** But all fat arrow functions are anonymous.

File: README.md [hunk 4]
```diff
@@ -61,6 +77,10 @@ var obj = {
 
 -- YOUR ANSWER HERE --
 
+- ES5
+- Is the function anonymous? NO
+- How do you call the function? ? add(3,4);
```
Comment (mesprague) [inline README.md:87]: ```obj.add(5,9);``` is how you would call this. But I think it was written ambiguously.

File: README.md [hunk 5]
```diff
@@ -85,11 +109,15 @@ var obj = {
 
 -- YOUR ANSWER HERE --
 
+- ES5
+- Is the function anonymous? NO
+- How do you call the function? obj;
```
Comment (mesprague) [inline README.md:119]: ```obj.add(3243,2131);```

File: README.md [hunk 6]
```diff
@@ -85,11 +109,15 @@ var obj = {
 
 -- YOUR ANSWER HERE --
 
+- ES5
```
Comment (mesprague) [inline README.md:117]: ES6. Look up ES6 Object Shorthand for method properties

File: README.md [hunk 7]
```diff
@@ -99,3 +127,4 @@ console.log(i)
 
 -- YOUR ANSWER HERE --
 
+Mo  ?
```
Comment (mesprague) [inline README.md:135]: Bob
Curly
1
Mo
Curly
2
2

Please experiment with for loops like Carlos did in class.

File: README.md (hunk 6 above)
Comment (taramason) [reply README.md:117]: Hey Matt, can you double check that this is actually ES6? I looked this up and can't figure out how this is ES6, still looks like ES5 to me. Are we talking about the problem: 
function multiply(x) {
  return x * y;
}

File: README.md (hunk 3 above)
Comment (taramason) [reply README.md:68]: okay

File: README.md (hunk 7 above)
Comment (taramason) [reply README.md:135]: Will do. Major gap in knowledge... because I learned this so long ago...

File: README.md (hunk 5 above)
Comment (taramason) [reply README.md:119]: What is this called so I can practice?

**Pull request closed without merging**